SublimeLinter.sublime-settings

```diff
@@ -21,6 +21,15 @@
     // to an icon in the theme (they usually include "warning" and "error")
     "gutter_theme": "none",
 
+    // EXPERIMENTAL: See below `demote_while_editing`.
+    "highlights.time_to_idle": 5,
+
+    // EXPERIMENTAL: One of 'none', 'ws_regions', 'warnings'
+    // Essentially picks a predicate fn from `highlights.py` to hide some
+    // lint errors while editing (busy). These errors appear after
+    // `time_to_idle` seconds or immediately on save.
+    "highlights.demote_while_editing": "ws_regions",
```

Comment:
I'd say `hide_while_editing`, because that's easier to understand (the word "hide" is also used 2 lines earlier, so it would also be more consistent).

That `ws_regions` means "regions containing whitespace isn't super clear. 

I'll leave it to you if you want to bikeshed about this right now. It's ok with me if you want to merge this as is, because it's good stuff and it's clearly marked as experimental so people can expect things to change here.

Reply:
I already made these changes, but I revert and take 'demote' now. Currently, I actually hide these regions. But, we could also (if I were not that dumb with these coloring/scope stuff) de-emphasize them using a soft gray.

Reply:
Oh that’s also not a bad idea, using the `’comment’` scope for color when busy